cf/env_test.go

```diff
@@ -1,6 +1,7 @@
 package cf
```

Comment:
should this be adapted or we leave it like this?

Reply:
i will adapt it